test/contrib/s3_test.py

```diff
@@ -425,6 +424,26 @@ def test_copy_empty_file(self):
         """
         self._run_copy_test(self.test_put_multipart_empty_file)
 
+    @mock_s3
+    @skipOnTravis('https://travis-ci.org/spotify/luigi/jobs/145895385')
```

Comment:
can you elaborate?

Reply:
Sorry. Copy/paste. Will remove to try to green build without it.